sources/AppBundle/Indexation/Talks/Transformer.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace AppBundle\Indexation\Talks;
+
+use AppBundle\Event\Model\Event;
+use AppBundle\Event\Model\Planning;
+use AppBundle\Event\Model\Speaker;
+use AppBundle\Event\Model\Talk;
+
+class Transformer
+{
+    /**
+     * @param Planning $planning
+     * @param Talk $talk
+     * @param Event $event
+     * @param Speaker[]|\Traversable $speakers
+     *
+     * @return array
+     */
+    public function transform(Planning $planning, Talk $talk, Event $event, \Traversable $speakers)
+    {
+        $spakersLabels = [];
+        foreach ($speakers as $speaker) {
```

Comment:
est-ce qu'on pourrait n'itérer qu'une fois sur $speakers en regroupant les 2 boucles ?